### ISSUE TITLE
UnusedImports behaviour is different from documentation

### ISSUE THREAD

**Issue by chao1995** [checkstyle/checkstyle#3741]
/var/tmp $ javac Test.java

No output.

/var/tmp $ cat Test.java

```
 // Doesn't warn - incorrect
import java.lang.String;

public class Test {
    private static final String SOMETHING = "a string";
}
```

/var/tmp $ cat config.xml

```
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="UnusedImports" />
    </module>
</module>
```

/var/tmp $ java -jar checkstyle-7.4-all.jar -c config.xml Test.java

Starting audit...
Audit done.

---------------

Describe what you expect in detail.

--------------

As documented [in the third item of UnusedImports description](http://checkstyle.sourceforge.net/config_imports.html#UnusedImports), 
> The class imported is from the java.lang package. For example importing java.lang.String.

I expect the import of `java.lang.String` to be reported as unused imports, however, it doesn't.

I'm aware there is a [RedundantImport](http://checkstyle.sourceforge.net/config_imports.html#RedundantImport) which can perform this check correctly, and in documentation it is just a proper subset of UnusedImport, so maybe this issue is just because of **outdated documentation**.

**Comment by MEZk** [comment]
@romani 
Please. assign

**Comment by rnveach** [comment]
Fix is merged
